### PR TITLE
feat(fscomponents): Replace all occurrences of "any" in LoginFormFK component

### PR DESCRIPTION
**Pull request by IvanMatchin**
### Description
Replace all occurrences of "any" in LoginFormFK component

### CODE REVIEW

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
-  fieldsStyleConfig?: any; // the custom stylesheet we want to merge with the default stylesheet
-  onSubmit?: (value: any) => void; // the behaviour we want onpress of submit button
-  submitButtonStyle?: any;
-  submitTextStyle?: any;
-  submitText?: any; // Text to override the submit button
-  style?: any;
-  fieldsOptions?: any; // any extra desired behaviour, like placeholders
-  value?: any;
+  fieldsStyleConfig?: Dictionary; // custom stylesheet we want to merge with the default stylesheet
+  onSubmit?: <T>(value: T) => void; // the behaviour we want onpress of submit button
```
Comment (bweissbart) [inline packages/fscomponents/src/components/UpdateNameOrEmail.tsx:25]: Please make this `<T = any>` to preserve backwards compatibility.

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
+  submitButtonStyle?: StyleProp<ViewStyle>;
+  submitTextStyle?: StyleProp<ViewStyle>;
```
Comment (bweissbart) [inline packages/fscomponents/src/components/UpdateNameOrEmail.tsx:27]: This should be `StyleProp<TextStyle>`.

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx [hunk 3]
```diff
@@ -1,27 +1,34 @@
...
+  submitText?: () => void; // Text to override the submit button
```
Comment (bweissbart) [inline packages/fscomponents/src/components/UpdateNameOrEmail.tsx:28]: This should be type `string`.

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx [hunk 4]
```diff
@@ -1,27 +1,34 @@
...
+  style?: StyleProp<ViewStyle>;
+  fieldsOptions?: Dictionary; // extra desired behaviour, like placeholders
+  value?: string;
```
Comment (bweissbart) [inline packages/fscomponents/src/components/UpdateNameOrEmail.tsx:31]: Per the [tcomb-form-native docs](https://github.com/gcanti/tcomb-form-native#adding-a-default-value-and-listen-to-changes), value represents the current value of the form so this should be typed as Dictionary.

**Review by bweissbart — CHANGES_REQUESTED** [review]

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx (hunk 4 above)
Comment (IvanMatchin) [reply packages/fscomponents/src/components/UpdateNameOrEmail.tsx]: @bweissbart Resolved.

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx (hunk 2 above)
Comment (IvanMatchin) [reply packages/fscomponents/src/components/UpdateNameOrEmail.tsx]: @bweissbart Resolved.

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx (hunk 1 above)
Comment (IvanMatchin) [reply packages/fscomponents/src/components/UpdateNameOrEmail.tsx]: @bweissbart Resolved.

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx (hunk 2 above)
Comment (IvanMatchin) [reply packages/fscomponents/src/components/UpdateNameOrEmail.tsx]: @bweissbart Resolved

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx (hunk 3 above)
Comment (IvanMatchin) [reply packages/fscomponents/src/components/UpdateNameOrEmail.tsx]: @bweissbart Resolved

File: packages/fscomponents/src/components/UpdateNameOrEmail.tsx (hunk 4 above)
Comment (IvanMatchin) [reply packages/fscomponents/src/components/UpdateNameOrEmail.tsx]: @bweissbart Resolved

**Comment by IvanMatchin** [comment]
@bweissbart Updated

**Review by bweissbart — APPROVED** [review]

**Pull request merged**